docs/validator/keys.md

```diff
@@ -16,7 +16,7 @@ There are 3 types of key pairs on the NEAR platform
 - validator keys
 - node keys
 
-**Signer keys** are the ones we all know and love.  They're used by all accounts on the network to sign transactions like `sendMoney` and `stake` before sending them to the network.  Signer keys are not related to running a node in any way.  End users who sign up through the [NEAR Wallet](https://wallet.testnet.nearprotocol.com/) get their own signer keys, for example. These are the keys that humans think about and keep safe.
+**Signer keys** are the ones we all know and love.  They're used by all accounts on the network to sign transactions like `sendMoney` and `stake` before sending them to the network.  Signer keys are not related to running a node in any way.  End users who sign up through the [NEAR Wallet](https://wallet.betanet.nearprotocol.com/) get their own signer keys, for example. These are the keys that humans think about and keep safe.
```

Comment:
please update this URL, we just merged a PR that moves `nearprotocol.com` to `near.org`